Model/Config.php

```diff
@@ -510,4 +510,15 @@ public function getMagentoVersion(): string
     {
         return $this->productMetadata->getVersion();
     }
+
+    /**
+     * Get the current Git commit hash.
+     *
+     * @return string The current Git commit hash.
+     */
+    public function getGitCommit(): string
+    {
+        $pluginPath = escapeshellarg(__DIR__);
+        return trim(shell_exec("cd $pluginPath && git rev-parse HEAD"));
```

Comment:
what if git is not installed on the system?

Reply:
will return null and its covered.